.gitignore

```diff
@@ -7,6 +7,8 @@
 *~
 *.swp
 dmoj/local_settings.py
-resources/style.css
+resources/dark
 resources/martor-description.css
+resources/style.css
+resources/vars.scss
```

Comment:
Why this line?

Reply:
Because of `cp resources/vars-$1.scss resources/vars.scss`. Also, the user can kill the script while `vars.scss` exists, and adding `vars.scss` to the gitignore covers this case.